apps/ensadmin-next/src/components/connections/ensnode-url-validator.ts

```diff
@@ -0,0 +1,26 @@
+export interface EnsNodeValidator {
+  validate(url: string): Promise<{ isValid: boolean; error?: string }>;
+}
+
+export class BasicEnsNodeValidator implements EnsNodeValidator {
```

Comment:
This is a basic impl, to be replaced with more advanced compatibility check, as per:
- #319